src/seedu/addressbook/ui/TextUi.java

```diff
@@ -87,6 +87,18 @@ public String getUserCommand() {
         showToUser("[Command entered:" + fullInputLine + "]");
         return fullInputLine;
     }
+    
+    /** Shows a list of strings to the user, formatted as an indexed list. */
+    private void showToUserAsIndexedList(List<String> list) {
+        showToUser(Formatter.getIndexedListForViewing(list));
+    }
+    
+    /** Shows message(s) to the user */
+    public void showToUser(String... message) {
+        for (String m : message) {
+            out.println(LINE_PREFIX + m.replace("\n", LS + LINE_PREFIX));
+        }
+    }
 
```

Comment:
Perhaps more formatting-related stuff can be moved to the Formatter? May be even things like this should be moved to the Formatter? Note that it may not be clear if something belongs in the Formatter class or the TextUi class. It depends on how you define the responsibility of the Formatter class.